common/src/main/java/com/wynntils/utils/type/ErrorOr.java

```diff
@@ -41,6 +43,15 @@ public boolean hasError() {
         return error != null;
     }
 
+    public <T> ErrorOr<T> logged() {
```

Comment:
This was a very elegant solution!

If we start to use this in other contexts, we can probably remove the throw clause and just do a no-op if there is no error.